geometry_msgs/Twist stiffness
geometry_msgs/Twist damping